### DISCUSSION TITLE
Issue

### DISCUSSION THREAD

**Discussion by yvv99** [termux/termux-packages#7254] [Q&A] [answered]
My terminal is lagging . When i try to type something it wasn't showing before i press spacebar and when I'm trying to cancel any command its also not done.
Exit command is also not work .
I already uninstalled twice but nothing happens but my all packages are deleted then also .

**Comment by agnostic-apollo** [comment, marked as answer]
Set `enforce-char-based-input=true`

https://wiki.termux.com/wiki/Terminal_Settings#Workaround_for_some_keyboards_issues
